docroot/.htaccess

```diff
@@ -93,8 +93,9 @@ AddEncoding gzip svgz
   RewriteCond %{REQUEST_URI} node_modules [OR,NC]
   RewriteCond %{REQUEST_URI} ^/vendor [OR,NC]
   RewriteCond %{REQUEST_URI} "/wp-(admin|content/plugins/|includes|cron\.php|config\.php|login\.php|signup\.php)|xmlrpc.php" [OR,NC]
-  RewriteCond %{THE_REQUEST} \.php[/\s?] [OR,NC]
-  RewriteCond %{THE_REQUEST} \.html[/\s?] [NC]
```

Comment:
If I had to guess, based on c8babb1264347394ec71aa0ecf08a1aef27ca800, I would imagine this was put in place to block malicious bot traffic that was probing for vulnerabilities. Probably a thought process along the lines of "there are no legitimate requests for paths ending in `.php` or `.html`, so let's not even let them get to Drupal." But now you've seen a use case where this causes a problem. Would have to ask Mike to know for sure.